sciencebeam_gym/preprocess/preprocessing_utils.py

```diff
@@ -75,6 +77,13 @@
   PdfToPng
 )
 
+# deprecated, moved to sciencebeam_gym.utils.file_path
```

Comment:
to my knowledge I changed all internal references but as this project is also used as a utility project those methods may also be used by other projects